PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResponse.kt

```diff
@@ -0,0 +1,46 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import com.braintreepayments.api.BrowserSwitchOptions
+import com.braintreepayments.api.core.ExperimentalBetaApi
+
+@ExperimentalBetaApi
+sealed class PayPalVaultEditResponse {
+
+    /**
+     * The PayPal vault edit flow completed successfully.
+     *
+     * @property riskCorrelationId This ID is used to link subsequent retry attempts if payment is declined
+     */
+    class ReadyToLaunch internal constructor(
+        val riskCorrelationId: String,
+        val response: EditFIAgreementSetup,
+        internal var browserSwitchOptions: BrowserSwitchOptions? = null
+    ) : PayPalVaultEditResponse()
+
+    /**
+     * There was an [error] in the PayPal vault edit flow.
+     */
+    class Failure internal constructor(
+        val error: Exception,
+        val riskCorrelationId: String? = null,
```

Comment:
I don't think we need the `riskCorrelationId` on the `Failure` case - usually we just return the error